_pages/en/strategy-open-first.md

```diff
@@ -0,0 +1,445 @@
+
+### Secure
+
+> We make sure the code we use and the code we share is free from vulnerabilities by applying continuous security testing.
+
+Following IT security best practices and monitoring security advisories, automated continuous security testing will be in place to make sure that OSS components that we use in our applications are free from vulnerabilities.
+Likewise, we will thoroughly test the code that we intend to share.
+
+ESDC already has experience in this area in our security teams and the Computer Emergency Response Team for the institutions (CERT-EU), and from EU-FOSSA/EU-FOSSA2.
+
+Notes:
+
+Security and privacy methods and tools should be made available for all to ensure that this practice can be accomplished with the least barriers as possible.
+
+### Stay in control
+
+> We promote open standards and specifications that are implemented and distributed in OSS as a way to avoid vendor lock-in.
+
+Interoperability is of utmost importance to the ESDC and the GC.
+The key to interoperability is the use of well-established standards and specifications.
+To ensure our digital sovereignty and guarantee a level playing-field, for all future IT developments ESDC will promote standards and specifications that are implemented and distributed through OSS, and include this in its corporate governance approach.
+
+## 6. Implementing the Strategy
+
+The GC is an enthusiastic user of and contributor to free and OSS.
+This "Open First" strategy will ensure that ESDC becomes even more involved.
+
+### Current situation and lessons learned
+
+TODO: OSS Framework has identified current state as (see Framework for more details) Mainly OSS is in use already but before Framework wasn't clear on how to maintain best practices. Framework approved but not implemented.
+
+### The right balance
+
+This strategy comes with an action plan that details how ESDC will encourage and leverage open standards and OSS development practices to co-create, share and reuse across the organisation.
+In addition to this internal focus on an open working culture, the strategy and action plan provide for increasing outreach to open standards and OSS communities and will help ESDC become a strong force in the community of Canadian public services actively involved in open standards and OSS.
+
+With these internal and external activities, the strategy and action plan constitute practical instruments to help achieve the digital transformation of ESDC.
+
+### Open First Center of Enablement (CoE)
+
+This strategy is firmly rooted in the ESDC's and the wider GC policy context.
+For it to succeed, a proper enabling environment must be put in place to guide, encourage and nudge change across the organisation.
+
+The strategy and the action plan are inextricably linked.
+The pivotal point will be an Open First CoE that enjoys genuine political and organisational support, involving all directorates-general.
```

Comment:
do we mean all directorates-general across ESDC or just within IITB?

Reply:
replaced "directorates-general" with "branches within ESDC" in 3 places